system/iptables_raw.py

```diff
@@ -304,6 +321,12 @@ def _check_compatibility(self):
         else:
             Iptables.module.fail_json(msg="Could not fetch iptables version! Is iptables installed?")
 
+    # Check if /etc/init.d/iptables-persistent for older Debian based systems
+    # Check if /etc/init.d/netfilter-persistent for newer Debian based systems
+        if self.is_debian():
+            if not os.path.isfile('/etc/init.d/netfilter-persistent') and not os.path.isfile('/etc/init.d/iptables-persistent'):
+                Iptables.module.fail_json(msg="Could not detect iptables-persistent package. Is iptables-persistent installed?")
```

Comment:
From what I read, if you do not use [iptables-persistent](https://www.thomas-krenn.com/en/wiki/Saving_Iptables_Firewall_Rules_Permanently), you will than need to update the pre and post scripts with this data, and I think this init script is the path of least resistance. If you think this route is not necessary, than if you want you can update pre and post  startup scripts as per this document [iptables_ubuntu](https://help.ubuntu.com/community/IptablesHowTo).